lexibank_lieberherrkhobwa.py

```diff
@@ -36,92 +45,64 @@ def cmd_download(self, **kw):
-            # concept id in the dataset
-            concept_by_index = {}
-            for cid, concept in enumerate(self.conceptlist.concepts.values()):
-                # We can get the concept id from the raw data
-                concept_by_index[1+(cid*2)] = concept.id
-
-                # Add the concept
-                ds.add_concept(
-                    ID=concept.id,
-                    Name=concept.english,
-                    Concepticon_ID=concept.concepticon_id,
-                    Concepticon_Gloss=concept.concepticon_gloss,
-                )
-
-            # Read raw languages and add to the dataset; at the same time,
-            # build a map that can be used for iterating over language ids
-            langs = {
-                l['Source_Name']: {'id': l['ID'], 'source': l['Source']}
-                for l in self.languages
-            }
-            ds.add_languages()
-
-            # iterate over the source adding lexemes and collecting cognates
-            for row in data[2:]:
-                # Confirm the row refers to one of the languages to be
-                # included (i.e., no reconstructions)
-                if row[0] in langs:
-                    for cid in range(1, len(row), 2):
-                        # Skip over rows with empty fields for cogid
-                        if not row[cid+1]:
-                            continue
-
-                        # Compute a cognate_id number; lingpy now requires
-                        # this to be an integer
-                        cognate_id = cid * 100 + int(row[cid + 1])
-
-                        # Extract the value from the raw data, skipping over
-                        # missing or non-existing forms
-                        value = row[cid].strip()
-                        if value != "NA":
-                            for lex in ds.add_lexemes(
-                                    Language_ID=langs[row[0]]['id'],
-                                    Parameter_ID=concept_by_index[cid],
-                                    Value=value,
-                                    Cognacy=cognate_id,
-                                    Source=langs[row[0]]['source'],
-                            ):
-                                ds.add_cognate(
-                                    lexeme=lex,
-                                    Cognateset_ID=cognate_id,
-                                    Source="Lieberherr2017",
-                                    Alignment_Source="",
-                                )
+    def cmd_makecldf(self, args):
+        # Add bibliographic sources
+        args.writer.add_sources()
+
+        # Read raw concept data and add to dataset; at the same time,
+        # build a map between the concept index as used in data and the
+        # concept id in the dataset
+        concept_lookup = {}
+        for cid, concept in enumerate(self.conceptlists[0].concepts.values()):
```

Comment:
Sorry for annoying you here, but can I ask you to stick to our general ID handling for concepts, as we practiced it in the other lists?

```python
concept_lookup[1+(cid*2)] = concept.id.split('-')[-1]+'_'+slug(concept.english)
```